doc/architecture/decisions/0008-concept-of-bedspaces.md

```diff
@@ -0,0 +1,62 @@
+# 8. concept-of-bedspaces
+
+Date: 2023-02-28
+
+## Status
+
+Accepted
+
+## Context
+
+CAS1 and CAS3 share an API which includes the modelling for premises, rooms and
+beds. This modelling was initially set out by CAS1 and CAS3 have been working to
+extend in order to meet the needs of Temporary Accommodation (CAS3).
+
+The relationships are straight forward. A premises can have many rooms and a
+room can have many beds.
+
+For CAS1 they will often have the scenario of multiple beds belonging to a single
+room.
+
+For CAS3 we only expect one bed per room for the foreseeable future however
+cannot guarantee that this will always remain the case.
+
+At the time of writing an individual bed record doesn't itself contain any data,
+this too may change in future.
+
+Rooms serve to group related beds and to encapsulate shared attributes such as
+physical characteristics of the space. These are used in the matching journey
+where the risks and needs of person leaving prison are matched up to a suitable
+premises and room.
+
+Both teams have the need to help users create bookings for physical beds and to
+report on occupancy levels based on bed, rather than room. This points us
+towards using the existing bed model as the smallest unit of data for both
+services.
+
+Given the CAS3 team have 1 bed per room, we wanted to tailor the user
+experience to avoid unnecessary clicks. Instead of asking the user to navigate
+from premises through to rooms and on to the bed we can instead present all beds
+in a premises alongside their room characteristics. This removes the concept of
+'rooms' from our user.
+
+Finally, our users were more familiar with the language of 'bedspace' over 'bed'.
+
+## Decision
+
+Temporary Accommodation will introduce the concept of a "bedspace" to users.
+
+## Consequences
```

Comment:
It's maybe a bit fluffy as a consequence but one that I think is worth considering/documenting is that this decision probably causes a divergence between the concepts used by non-technical vs technical team members to think about and talk about the service. As in, technical team members kind of have to think about premises -> room -> bed -> booking, whereas non-technical people don't think about / talk about rooms, or maybe even know what they are?

Clearly this is a general issue with two services sharing an API, and also comes up in the case of Voids, which in our team are called voids, but in the code are called lostBeds right up until they are rendered in nunjucks.

Reply:
I think that's fair. There is a cognitive burden for us to translate bedspaces into what they really represent and then back in a language the rest of the team can relate to. As you say that's also true of voids/lost beds, perhaps also worth an ADR? 

I'll add another point though, though the overhead could be implied there's no harm in making it explicit.